### ISSUE TITLE
TensorFlow 2.5 적용과 한계

### ISSUE THREAD

**Issue by quantylab** [quantylab/rltrader#109]
TensorFlow 2에서는 Graph를 생성하지 않고 코드를 [즉시 실행(Eager Execution)](https://www.tensorflow.org/guide/eager) 합니다.
이로 인해 Graph, Session 등을 신경쓰지 않아도 되서 코드를 쉽게 작성할 수 있습니다.

> 사실 케라스를 사용하면 큰 차이가 없습니다만...

그런데 TF2의 Eager Execution에 의해 predict를 많이 호출하는 RLTrader의 속도가 매우 느려집니다.

결국 최신 TF2를 사용하면서 TF2 기능을 끄도록 했습니다. https://github.com/quantylab/rltrader/blob/dev/networks.py#L12

**Comment by kyle-109** [comment]
원인을 찾아주셔서 감사합니다! 그러면 새로운 TF2버전을 사용해도 TF1버전과 동일한 속도로 코드가 돌아갈까요?

**Comment by quantylab** [comment]
@kyle-109 네 TF2를 사용하지만 속도 저하의 원인인 Eager Execution을 비활성화하여 TF1과 같은 방식으로 작동합니다.
